src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 import tensorflow as tf
 from openvino.frontend.tensorflow.py_tensorflow_frontend import _FrontEndDecoderBase as DecoderBase
-from openvino.runtime import PartialShape, Type, OVAny, Tensor
+from openvino._pyopenvino import PartialShape, Type, OVAny, Tensor
```

Comment:
please, don't import from the lib directly -- it was marked intentionally for internal usage.
Please, add imports of these classes from openvino._pyopenvino to openvino.__init__